core/api/core.api

```diff
@@ -1,4 +1,4 @@
-public final class io/shortway/kobankat/CustomerInfo_androidKt {
+public final class com/revenuecat/purchases/kmp/CustomerInfo_androidKt {
```

Comment:
I agree that in theory we wouldn't need manual API tests. However, I added them because of these issues which they reproduce: https://github.com/RevenueCat/purchases-kmp/issues/48, https://github.com/RevenueCat/purchases-kmp/issues/51. This was only found by calling a method/field and compiling that call. 

The underlying issue here is an extension shadowing a member, but changing its type. I'm planning to catch those issues with a Detekt rule [SDK-3464](https://linear.app/revenuecat/issue/SDK-3464). So if we have that rule, then we might consider removing the manual API tests.

Reply:
Nice! Thanks for the explanation!